ui/partials/main/thWatchedRepoNavPanel.html

```diff
@@ -38,16 +38,16 @@
                     </span>
                 </span>
 
-                <!--Search Field-->
-                <span ng-controller="SearchCtrl" class="form-group form-inline" id="platform-job-text-search-field-parent">
-                    <input id="platform-job-text-search-field"
-                           title="Click to enter filter values"
+                <!--Quick Filter Field-->
+                <span ng-controller="SearchCtrl" class="form-group form-inline" id="quick-filter-parent">
+                    <input id="quick-filter"
+                           title="Click to enter quick filter values"
                            ng-model="searchQueryStr" ng-keydown="search($event)" type="text"
                            class="form-control input-sm" required
-                           placeholder="Filter platforms & jobs"
+                           placeholder="Quick filter"
```

Comment:
I had actually mis-read this on first pass when you asked, this `placeholder` line doesn't need to change either, sorry. :)